src/Uno.UI.RuntimeTests/MUX/Utilities/TestHelpers.cs

```diff
@@ -163,7 +163,8 @@ public static async Task ClearVisualTreeRoot()
 		{
 			await TestServices.WindowHelper.WaitForIdle();
 
-			UnoAutoResetEvent unloadedEvent = new UnoAutoResetEvent(false);
+			// Uno TODO: CI gets flaky with this. However, the flakiness isn't reproducible locally so far.
+			//UnoAutoResetEvent unloadedEvent = new UnoAutoResetEvent(false);
```

Comment:
This mostly reverts the effect of the fix for awaiting TestCleanup (https://github.com/unoplatform/uno/pull/16650). This is just a hack to get CI stable, but we should investigate why it's happening at some point.

I'll kick CI few times in the PR just to make sure it's more stable. Keeping draft until I confirm this change makes it more stable